### PR TITLE
Video.js 7.4 blogpost

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Comment by gkatsev** [comment]
Deploy preview for *videojs-blog* failed.

Built with commit 2a073e9b89ecbd21b8f18a8ed574dd5285e0584d

https://app.netlify.com/sites/videojs-blog/deploys/5c3909cfeb15ca00086883b0

File: source/_drafts/Video-js-7-4.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+tags:
+  - a11y
+  - responsive
+  - languages
+  - live-ui
+  - focus-visible
+---
+
+It's time to have an overview of Video.js 7.4, first released early December.
+The big new feature for this release is a UI that allows you to seek during live streams.
```
Comment (ldayananda) [inline source/_drafts/Video-js-7-4.md]: I would make this sentence and the one below bullet points to read a bit easier

File: source/_drafts/Video-js-7-4.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+We updated focus-visible to work with our Menus, added more translations, added a replay option to the Play/Pause button, and many many fixes, multiple of which are accessibility related.
```
Comment (ldayananda) [inline source/_drafts/Video-js-7-4.md]: many, many fixes

Comment (ldayananda) [reply source/_drafts/Video-js-7-4.md]: also might want to link to docs on Menus

File: source/_drafts/Video-js-7-4.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+We also dropped our usage of Grunt from our build process.
+We owe a lot to grunt as it has served us well but it was time to move on.
```
Comment (ldayananda) [inline source/_drafts/Video-js-7-4.md]: I would remove the newline here

File: source/_drafts/Video-js-7-4.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+- [@vitaliytv](https://github.com/vitaliytv) (first time contributor!)
+- [@oaprograms](https://github.com/oaprograms) (first time contributor!)
+- [@xjoaoalvesx](https://github.com/xjoaoalvesx) (first time contributor!)
+- [@webdeveloperpr](https://github.com/webdeveloperpr) (first time contributor!)
+- [@smbea](https://github.com/smbea) (first time contributor!)
+- [@dustin71728](https://github.com/dustin71728) (first time contributor!)
+
+## Live UI
+Video.js has supported live streams for a while, either natively, or via [videojs-http-streaming (VHS)](https://github.com/videojs/http-streaming).
+However, this UI was very minimal, it disabled the progress bar and basically just allowed paused, though, there was an indicator that this was a live stream.
```
Comment (ldayananda) [inline source/_drafts/Video-js-7-4.md]: allowed pausing?

File: source/_drafts/Video-js-7-4.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+
+![Video.js playing a live stream](live ui.png)
```
Comment (ldayananda) [inline source/_drafts/Video-js-7-4.md]: it wasn't super clear if this is the new or old UI

File: source/_drafts/Video-js-7-4.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+- Russian (ru)
+- Welsh/Cymraeg (cy)
+- Ukrainian (uk)
+- Serbian (sr)
+- Swedish (sv)
+
+## Accessibility
+As always, we aim to make Video.js as accessible and usable as we can. To that end, we've had a bunch of accessibility related fixes in these releases.
+
+- time displays accessibility with VoiceOver
```
Comment (ldayananda) [inline source/_drafts/Video-js-7-4.md]: TimeDisplay?

File: source/_drafts/Video-js-7-4.md (hunk 3 above)
Comment (misteroneill) [reply source/_drafts/Video-js-7-4.md]: Also, capitalize "grunt" here.

File: source/_drafts/Video-js-7-4.md [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+- [@smbea](https://github.com/smbea) (first time contributor!)
+- [@dustin71728](https://github.com/dustin71728) (first time contributor!)
+
+## Live UI
+Video.js has supported live streams for a while, either natively, or via [videojs-http-streaming (VHS)](https://github.com/videojs/http-streaming).
+However, this UI was very minimal, it disabled the progress bar and basically just allowed paused, though, there was an indicator that this was a live stream.
+
+![Video.js playing a live stream](live ui.png)
+
+The new UI, looks pretty similar to the previous live UI and the regular control bar.
```
Comment (misteroneill) [inline source/_drafts/Video-js-7-4.md]: No comma needed.

File: source/_drafts/Video-js-7-4.md [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+The "live" indicator moves to the right side of the progress bar and also indicates whether we are playing back live or we are behind live.
+Clicking this button will seek to the "live point".
+The Time Displays and Tooltips will show time with respect to the live point, so, current time at the live point will be `0` and if you seek back 30 seconds, it'll show `-00:30`.
```
Comment (misteroneill) [inline source/_drafts/Video-js-7-4.md]: Should "Time Displays" and "Tooltips" be capitalized? Doesn't seem like they are proper nouns.

Also, `, so,` should be `; so,`. Don't think you need the comma before `it'll`.

**Review by misteroneill — COMMENTED** [review]
Overall, looks good. I think that some of the formatting makes it hard to read, though. Namely, it looks like each sentence is its own line, but this does not translate neatly into paragraphs when passed through Markdown. It might be better to not put newlines inside groups of sentences that are intended to be paragraphs.

**Comment by gkatsev** [comment]
yeah, I keep forgetting that this doesn't merge new lines into a paragraph. I'll remove the new lines before publishing, it does make reviewing slightly easier, though.

File: source/_drafts/Video-js-7-4.md (hunk 1 above)
Comment (gkatsev) [reply source/_drafts/Video-js-7-4.md]: we have a more details bullet point list below, was going to keep this as text.

**Review by brandonocasey — APPROVED** [review]
Looks good to me

**Review by misteroneill — APPROVED** [review]

**Pull request merged**